server/api/user/user.controller.js

```diff
@@ -65,7 +74,7 @@ export function create(req, res, next) {
       var token = jwt.sign({ _id: user._id }, config.secrets.session, {
         expiresIn: 60 * 60 * 5
       });
-      createVerificationToken(req, user);
+      createVerificationToken(req, user, (message) => {Sendgrid.signup(message)} );
```

Comment:
brackets aren't necessary with arrow syntax if it's a single expression (unless you don't want the return value)